roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: "StackStorm_{{ st2_pkg_repo }}"
+    baseurl: https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/el/{{ ansible_distribution_major_version }}/$basearch
+    repo_gpgcheck: yes
+    gpgkey: "https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/gpgkey"
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire: 300
+    gpgcheck: no
+    enabled: yes
+    sslverify: yes
+
+# Fixes "Failure talking to yum: Cannot retrieve repository metadata (repomd.xml) for repository: StackStorm_stable. Please verify its path and try again" when installing st2
+- name: Update ca-certificates package
+  become: yes
+  package:
+    name: ca-certificates
+    state: latest
```

Comment:
> [ANSIBLE0010] Package installs should not use latest

The `ansible-lint` warning is valid. Use `state: installed` where possible.
Installing `latest` package version on every playbook re-run is generally a bad practice (especially when ansible works in `ansible-pull` mode).

For `st2` we use `latest` intentionally as option so user can specify `st2_version=latest` to have latest StackStorm. 
Probably we should change default behavior there someday later since it could be dangerous for default installs.

For everything else the safe way is to upgrade software on the server by running specific playbook which focuses on OS package upgrades (or patches) to have full control and react accordingly if something went wrong.